queue_services/entity-filer/src/entity_filer/config.py

```diff
@@ -128,6 +128,11 @@ class _Config():  # pylint: disable=too-few-public-methods
     MINIO_BUCKET_BUSINESSES = os.getenv('MINIO_BUCKET_BUSINESSES', 'businesses')
     MINIO_SECURE = True
 
+    # determines which year of NAICS data will be used to drive NAICS search
+    NAICS_YEAR = int(os.getenv('NAICS_YEAR', '2017'))
+    # determines which version of NAICS data will be used to drive NAICS search
+    NAICS_VERSION = int(os.getenv('NAICS_VERSION', '3'))
+
```

Comment:
won't need this since we will be doing a REST call to GET NAICS code endpoint which only requires the NAICS code as input